packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -259,6 +277,15 @@ class FlutterErrorDetails extends Diagnosticable {
   /// This won't be called if [stack] is null.
   final IterableFilter<String> stackFilter;
 
+  /// A object that is responsible for causing this error.
+  ///
+  /// This property must be in a type that at least one describer in
```

Comment:
"must" may be a bit strong. maybe "The property will only be used in the output if at least one described in [offendingContextDescribers] can do something with it"?